shell/platform/cpp/client_wrapper/README

```diff
@@ -0,0 +1,9 @@
+This code is intended to be built into plugins and applications to provide
```

Comment:
I didn't see a unittest target for this. If there are not currently available, please create a stub target so folks know where to begin.

Reply:
So far FDE didn't have any tests :( Added test targets for the client wrappers, some infrastructure for writing non-trivial tests, and a couple of starter tests. It'll need a bunch of test coverage as follow-up.